packages/ERTP/src/types.js

```diff
@@ -350,6 +350,24 @@
  *
  * @property {(amount: Amount) => Payment} withdraw
  * Withdraw amount from this purse into a new Payment.
+ *
+ * @property {IssuerClaim} claim
+ * Like `issuer.claim`, but also associates the new payment with this purse's
+ * recovery set, so the payment can be recovered in emergencies.
+ *
+ * @property {() => CopySet<Payment>} getRecoverySet
```

Comment:
this does make purses non-robust servers in that any holder of a purse can impair the use by any other holder of a purse. I think that's OK , but it is a change.

Reply:
(we discussed and agreed it is ok. Because anyone holding a purse can withdraw everything from it, we encourage a programming pattern where a single purse is never help my multiple mutually suspicious parties, or even by multiple non-coordinated parties. Within that framing, we are not introducing any new vulnerabilities.)